src/mavsdk/core/mavlink_request_message_handler.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <array>
+#include <functional>
+#include <mutex>
+#include <optional>
+#include <vector>
+#include "mavlink_include.h"
+#include "mavlink_command_receiver.h"
+
+namespace mavsdk {
+
+class SystemImpl;
+
+class MavlinkRequestMessageHandler {
+public:
+    MavlinkRequestMessageHandler() = delete;
+    explicit MavlinkRequestMessageHandler(SystemImpl& system_impl);
+    ~MavlinkRequestMessageHandler();
+
+    using Params = std::array<float, 5>;
+    using Callback = std::function<std::optional<MAV_RESULT>(Params)>;
```

Comment:
Consider `const Params&` instead of `Params` here.

Reply:
Yes, would make sense. Thx.

Reply:
Changed in 71ff6ae5, finally.